# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1518415647 18000
#      Mon Feb 12 01:07:27 2018 -0500
# Node ID 5c266e59ebb997dabc255c647d077715f829b0eb
# Parent  3695c2cd69b85fef6344b017ddea3372ed8e8343
don't include "singleton-cleanup.h" in files that do not need it

* oct-stream.cc, sighandlers.cc, file-ops.cc: Don't include
singleton-cleanup.h.

diff --git a/libinterp/corefcn/oct-stream.cc b/libinterp/corefcn/oct-stream.cc
--- a/libinterp/corefcn/oct-stream.cc
+++ b/libinterp/corefcn/oct-stream.cc
@@ -40,17 +40,16 @@ along with Octave; see the file COPYING.
 #include "Array.h"
 #include "Cell.h"
 #include "byte-swap.h"
 #include "lo-ieee.h"
 #include "lo-mappers.h"
 #include "lo-utils.h"
 #include "oct-locbuf.h"
 #include "quit.h"
-#include "singleton-cleanup.h"
 #include "str-vec.h"
 
 #include "error.h"
 #include "errwarn.h"
 #include "input.h"
 #include "interpreter.h"
 #include "octave.h"
 #include "oct-iostrm.h"
diff --git a/libinterp/corefcn/sighandlers.cc b/libinterp/corefcn/sighandlers.cc
--- a/libinterp/corefcn/sighandlers.cc
+++ b/libinterp/corefcn/sighandlers.cc
@@ -34,17 +34,16 @@ along with Octave; see the file COPYING.
 #  define WIN32_LEAN_AND_MEAN
 #  include <windows.h>
 #endif
 
 #include "child-list.h"
 #include "cmd-edit.h"
 #include "oct-syscalls.h"
 #include "quit.h"
-#include "singleton-cleanup.h"
 #include "signal-wrappers.h"
 
 #include "bp-table.h"
 #include "defun.h"
 #include "error.h"
 #include "input.h"
 #include "interpreter-private.h"
 #include "interpreter.h"
diff --git a/liboctave/system/file-ops.cc b/liboctave/system/file-ops.cc
--- a/liboctave/system/file-ops.cc
+++ b/liboctave/system/file-ops.cc
@@ -41,17 +41,16 @@ along with Octave; see the file COPYING.
 #include "dir-ops.h"
 #include "file-ops.h"
 #include "file-stat.h"
 #include "gen-tempname-wrapper.h"
 #include "oct-env.h"
 #include "oct-locbuf.h"
 #include "oct-passwd.h"
 #include "quit.h"
-#include "singleton-cleanup.h"
 #include "stat-wrappers.h"
 #include "str-vec.h"
 #include "unistd-wrappers.h"
 
 // The following tilde-expansion code was stolen and adapted from
 // readline.
 
 // The default value of tilde_additional_prefixes.  This is set to
